### PR TITLE
Update readme to reflect stable install

### PR DESCRIPTION
**Pull request by sssangha**
Requirements file and mamba no longer translate to a stable build. Updating readme to reflect this

### CODE REVIEW

**Comment by dbekaert** [comment]
@sssangha Is there a reason we fall back to the past approach?
I thought mamba was getting more preferred as it resolves faster than conda?

**Comment by dbekaert** [comment]
@sssangha lets disable the test for the water mask?

**Comment by bbuzzanga** [comment]
> @sssangha Is there a reason we fall back to the past approach? I thought mamba was getting more preferred as it resolves faster than conda?

+1

**Comment by alexfore** [comment]
The issue here is that the README does not describe a working process to create a repeatable env in which to run this software. Specifically, the line `conda install mamba` does not work, presumably you also need to do something else (conda forge? I don't really know).  Even after using the instructions in the modified readme I am not able to obtain the same results as Simran did while running on another machine with an env he created.

Before I can implement any testing suites I actually need to be able to replicate Simran's results on my machine with my env, as clearly what I have now fails all my tests.  To do that we need a way to create envs for this software that work and give the same outputs when provided the same inputs and aria-tools software. Maybe your requirements file needs work to make this happen.

**Comment by sssangha** [comment]
@dbekaert and @bbuzz31 , I was able to confirm from Alex's suggestion that mamba works when specifying it can be found through `conda-forge`. I've updated the READme to be more explicit to reflect a stable installation that users can follow verbatim.

Will have to still figure out what's going wrong when using `requirements` vs the YML from scratch though.

**Comment by sssangha** [comment]
When the `jupyter_contrib_nbextensions` module was installed through pip originally, as opposed to through conda-forge, the following build installation error was raised:
```
(base) [ssangha@trappist ARIA-tools]$ mamba env create -f environment_ariatools_new2PRtest.yml
Retrieving notices: ...working... done
Channels:
 - conda-forge
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: done

Downloading and Extracting Packages:
                                                                                                                                                                                                                                
Preparing transaction: done                                                                                                                                                                                                     
Verifying transaction: failed                                                                                                                                                                                                   
                                                                                                                                                                                                                                
CondaVerificationError: The package for jsonschema-specifications located at /u/trappist-r0/ssangha/conda_installation/stable_april1_2024/pkgs/jsonschema-specifications-2023.12.1-pyhd8ed1ab_0                                 
appears to be corrupted. The path 'site-packages/jsonschema_specifications/schemas/draft201909/vocabularies/core'                                                                                                               
specified in the package manifest cannot be found.                                                                                                                                                                              
                                                                                                                                                                                                                                
CondaVerificationError: The package for jsonschema-specifications located at /u/trappist-r0/ssangha/conda_installation/stable_april1_2024/pkgs/jsonschema-specifications-2023.12.1-pyhd8ed1ab_0                                 
appears to be corrupted. The path 'site-packages/jsonschema_specifications/schemas/draft202012/vocabularies/core'                                                                                                               
specified in the package manifest cannot be found.                                                                                                                                                                              
                                                                                                                                                                                                                                
ClobberError: This transaction has incompatible packages due to a shared path.                                                                                                                                                  
  packages: conda-forge/linux-64::libkml-1.3.0-h01aab08_1018, conda-forge/linux-64::minizip-4.0.5-h0ab5242_0                                                                                                                    
  path: 'include/minizip/ioapi.h'                                                                                                                                                                                               
                                                                                                                                                                                                                                
                                                                                                                                                                                                                                
ClobberError: This transaction has incompatible packages due to a shared path.                                                                                                                                                  
  packages: conda-forge/linux-64::libkml-1.3.0-h01aab08_1018, conda-forge/linux-64::minizip-4.0.5-h0ab5242_0                                                                                                                    
  path: 'include/minizip/unzip.h'                                                                                                                                                                                               
                                                                                                                                                                                                                                
                                                                                                                                                                                                                                
ClobberError: This transaction has incompatible packages due to a shared path.                                                                                                                                                  
  packages: conda-forge/linux-64::libkml-1.3.0-h01aab08_1018, conda-forge/linux-64::minizip-4.0.5-h0ab5242_0                                                                                                                    
  path: 'include/minizip/zip.h'                                                                                                                                                                                                 
                                                                                                                                                                                                                                
                                                                                                                                                                                                                                
ClobberError: This transaction has incompatible packages due to a shared path.                                                                                                                                                  
  packages: conda-forge/linux-64::libkml-1.3.0-h01aab08_1018, conda-forge/linux-64::minizip-4.0.5-h0ab5242_0                                                                                                                    
  path: 'lib/libminizip.so'
```

File: README.md [hunk 1]
```diff
@@ -67,12 +67,15 @@ Below we list the dependencies for ARIA-tools
...
 
 ### Conda
-Below we outline the different steps for setting up the ARIA-tools while leveraging Anaconda for installation of the requirements. Run the commands below to download/clone the ARIA-tools package to your local directory. It is advised to use `mamba` as prompted to speed the install up:
+Below we outline the different steps for setting up the ARIA-tools while leveraging Anaconda for installation of the requirements. Run the commands below to download/clone the ARIA-tools package to your local directory.:
 
 ```.tcsh
+conda config --add channels conda-forge
 conda install mamba
 git clone https://github.com/aria-tools/ARIA-tools.git
```
Comment (cmarshak) [inline README.md]: Do not install `mamba` through `conda`; it is not recommended: https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html#existing-conda-install-not-recommended

Install `miniforge` which comes built with `conda` and `mamba` both.

No need to add conda-forge channel either as that is now the default channel.

Comment (cmarshak) [reply README.md]: And if you don't believe me, believe Yunjun: https://github.com/yunjunz/conda-envs

**Review by cmarshak — CHANGES_REQUESTED** [review]

**Comment by sssangha** [comment]
@cmarshak , agreed went ahead and adjusted the readme to reflect your suggestion. Specifically, install the package using miniforge

File: README.md [hunk 2]
```diff
@@ -67,13 +67,17 @@ Below we list the dependencies for ARIA-tools
...
+
+__[Conda](https://docs.conda.io/en/latest/index.html)__ is a cross-platform way to use Python that allows you to setup and use "virtual environments," which allows for the easy installation and management of all of the required dependencies. We recommend using the [Miniforge](https://github.com/conda-forge/miniforge) conda environment manager, which uses conda-forge as its default code repo. Alternatively, see __[here](https://docs.anaconda.com/anaconda/install/)__ for help installing Anaconda and __[here](https://docs.conda.io/en/latest/miniconda.html)__ for installing Miniconda.
 
 ### Conda
-Below we outline the different steps for setting up the ARIA-tools while leveraging Anaconda for installation of the requirements. Run the commands below to download/clone the ARIA-tools package to your local directory. It is advised to use `mamba` as prompted to speed the install up:
+Below we outline the different steps for setting up the ARIA-tools while leveraging Anaconda for installation of the requirements. Run the commands below to download/clone the ARIA-tools package to your local directory.:
 
 ```.tcsh
```
Comment (alexfore) [inline README.md:86]: I noticed that these instructions don't mention other OS or machine architectures, how to set the installation directory, or mention other shells that users may be using.  Can you look into that?

**Review by alexfore — CHANGES_REQUESTED** [review]
Can you also fix this line:
```
setenv PATH ${PATH}:${PWD}/tools/ARIAtools
```
which should be 
```
setenv PATH ${PATH}:${PWD}/tools/bin
```

**Comment by sssangha** [comment]
> Can you also fix this line:
> 
> ```
> setenv PATH ${PATH}:${PWD}/tools/ARIAtools
> ```
> 
> which should be
> 
> ```
> setenv PATH ${PATH}:${PWD}/tools/bin
> ```

done

File: README.md (hunk 2 above)
Comment (sssangha) [reply README.md:86]: @alexfore ok, I made the appropriate clarifications. While we have tested this out on BASH and Mac OS, I hadn't done it formally for this particular release.

I feel though a small snippet on how to install and setup through Mac would be something appropriate to add once I do have a chance to test out on another PR. Sounds good

**Comment by alexfore** [comment]
Hi @sssangha, this looks good. However, I think we also need to add the `parallel` and `pytest` packages to the mamba install.

**Comment by sssangha** [comment]
> Hi @sssangha, this looks good. However, I think we also need to add the `parallel` and `pytest` packages to the mamba install.

Sure, done. I tested the new environment with these packages included and all looks good. The golden outputs I just shared for `ariaDownload` were successfully accessed using this environment.

**Pull request merged**